### DISCUSSION TITLE
toolbar icon for indent/outdent command

### DISCUSSION THREAD

**Discussion by Thomas-U** [vincode-io/Zavala#82] [Ideas]
That would make it easier to invoke this command while the writing cursor is active. 

Right now I have to click outside the entry, then besides the text to make the whole entry selected, then I have access to the right context menu which is otherwise the one for text.

**Comment by vincode-io** [comment]
Tab and shift-tab also work to indent and out debt.

**Reply by mry53** [reply]
...with an external keyboard (talking of iPad specifically).

**Comment by vincode-io** [comment]
I went ahead and added optional toolbar items to indent and outdent. They will be in the next Alpha release.

**Reply by mry53** [reply]
On Mac, with Expand/Collapse added to toolbar when Icon + text is set as the Show option the text labels are long. Suggest making them Expand All/Collapse All rather than Expand All in Outline/Collapse All in Outline

**Reply by mry53** [reply]
Sorry... this is not in the proper place...

**Reply by vincode-io** [reply]
Thanks for letting me know. I've shortened the toolbar names, but left the tooltip the same name. Hopefully people figure out that it applies to the whole outline.

The Issues tab at the top of this page is the best way to report bugs.  https://github.com/vincode-io/Zavala/issues
